### DISCUSSION TITLE
Music at the score panel after finishing a map stops after switching to a leaderboard score

### DISCUSSION THREAD

**Discussion by klingonski** [ppy/osu#16248] [Q&A]
This is what i mean by switching to a leaderboard score. After this the music theme that comes with a skin stops
![osu_2021-12-26_00-01-03](https://user-images.githubusercontent.com/96668608/147395122-ae3b0a99-0494-4e4a-891f-fd362293a941.jpg)

**Comment by bdach** [comment]
had been reported before at #12823, then fixed at #12851 and then reverted again with #13330. not touching this again unless there's definitive consensus that needs to be changed

**Reply by peppy** [reply]
I'd definitely classify the behaviour as we have it as broken. I thought #13330 only broke the sound effects playing, not the music track as well.

**Reply by bdach** [reply]
report mentions "music theme that comes with a skin stops", so i don't think this is talking about the actual audio track of the current beatmap - but it's difficult to say given that the OP contains a *picture* rather than, you know, a *video* with *audio*

**Reply by peppy** [reply]
right, so it's the `applause` not the music.
